pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -118,7 +110,6 @@ func NewNodeRouteController(
 		queue:                   workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(minRetryDelay, maxRetryDelay), "noderoute"),
 		installedNodes:          cache.NewIndexer(nodeRouteInfoKeyFunc, cache.Indexers{nodeRouteInfoPodCIDRIndexName: nodeRouteInfoPodCIDRIndexFunc}),
 		wireGuardClient:         wireguardClient,
-		proxyAll:                proxyAll,
```

Comment:
I didn't check the code, but I assume this was just not used? I guess it was needed at some point in the past.